cockatrice/src/client/network/spoiler_background_updater.cpp

```diff
@@ -1,6 +1,7 @@
 #include "spoiler_background_updater.h"
 
 #include "../../game/cards/card_database.h"
+#include "../../game/cards/card_database_manager.h"
 #include "../../main.h"
```

Comment:
is main.h still needed here? if it is no longer needed please remove these includes.

Reply:
Yes, without main.h a variety of things are missing their references (such as trayIcon).